src/common/ComponentWithOverlay.js

```diff
@@ -16,8 +16,8 @@ const styles = createStyleSheet({
     overflow: 'hidden',
   },
   'top-right': {
-    right: 0,
-    top: 0,
+    right: -10,
+    top: 10,
```

Comment:
Hmm, looking closer, I see that we use `<ComponentWithOverlay />` in some places other than this one. Making an adjustment here will affect how the overlay looks in those cases, too, and it might not look right.

Instead, could we have `ComponentWithOverlay` take an optional new prop `customOverlayStyle?: ViewStyleProp`, and put that at the end of the `overlayStyle` array:

```diff
  const overlayStyle = [
    styles.wrapper,
    styles.overlay,
    styles[overlayPosition],
    {
      minWidth: overlaySize,
      height: overlaySize,
      borderRadius: overlaySize,
      backgroundColor: overlayColor,
    },
+   customOverlayStyle,
  ];
```

Then in just the `CountOverlay` case, we can pass the appropriate `customOverlayStyle`, so that this specific thing is fixed but we don't potentially break things for `ComponentWithOverlay`'s other callers.

Reply:
Oops, I think I missed to check `<ComponentWithOverlay />` whether we are using somewhere else or not. Thanks you found 😊.
Sure, @chrisbobbe!, I'll add the new prop in the component as you said and show you the preview. Thanks 😊